Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,28 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    ps = formula.opt_prefix
+    a = "#{ps}/bin"
+    b = "#{ps}/lib"
+    c = "#{ps}/libexec"
+    d = "#{ps}/sbin"
+
+    if Dir["#{a}/*"].empty?
+      if Dir["#{b}/*"].empty?
+        if Dir["#{c}/*"].empty?
+          if Dir["#{d}/*"].empty?
+            problem <<-EOS.undent
+              The installation seems to be empty. Please ensure the prefix
+                 is set correctly and expected files are installed.
+                 The prefix configure/make argument may be case-sensitive.
+            EOS
+          end
+        end
+      end
+    end
+  end
```

Comment:
Another county heard from, but I might prefer `[bin lib sbin share].any? { |p| p.exist? && !p.empty? }` because it allows for e.g. Python module installs that only have things in `lib/python2.7/site-packages/foo` which we care about at homebrew-python. Is there anything that should only have files in `libexec` after a successful installation?